src/Symfony/Component/Ldap/Adapter/RenameEntryInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Symfony\Component\Ldap\Adapter;
+
+use Symfony\Component\Ldap\Entry;
+
+/**
+ * @author Kevin Schuurmans <kevin.schuurmans@freshheads.com>
```

Comment:
Maybe just add a mention that this interface will be deprecated for 4.0, and merged with `EntryManagerInterface`.